web/src/locust/locustfile.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/python
```

Comment:
Python may not located at this place on all platforms and distributions. Using `#!/usr/bin/env python` is more portable.